lib/apis/user_api_manager.dart

```diff
@@ -32,6 +32,21 @@ class UserApiManager extends BaseApiManager {
     }
   }
 
+  static Future<void> updateNotificationSettingApi(
+      bool notificationSetting, Function success, Function fail) async {
```

Comment:
rename notificationSetting to shouldEnableNotification

Reply:
done

Reply:
follow this signature
static Future<void> updateNotificationSetting(
      bool shouldEnableNotification, Function success, Function fail) async {

Reply:
done